simulator/src/main/cpp/ChirpBluetoothPacket.cpp

```diff
@@ -0,0 +1,111 @@
+#include "ChirpBluetoothPacket.h"
+#include "utils.h"
+#include <string>
+
+ChirpBluetoothPacket::ChirpBluetoothPacket()
+{
+}
+
+ChirpBluetoothPacket::ChirpBluetoothPacket(char packet[], size_t size)
+{
+    this->packetType = parse_int(packet[0]);
+    this->groupId = parse_int(packet[1]);
+    
+    char node_id[] = {packet[2], packet[3]};
+    this->nodeId = parse_2digit_int(node_id);
```

Comment:
Is this all that is needed for the bluetooth packet to work with the radio? If so I'd add a comment above like you did for the radio with the packet structure